docs/REST API/rest-api.md

```diff
@@ -3185,10 +3185,14 @@ This report includes a subset of host vitals, and simplified policy and vulnerab
     "updated_at": "2023-09-16T18:52:19Z",
     "os_version": "CentOS Linux 8.3.2011",
     "disk_encryption_enabled": true,
+    "failing_policies_count": 1,
+    "failing_critical_policies_count": 1, // Fleet Premium only
     "failing_policies": [
       {
         "id": 123,
         "name": "Google Chrome is up to date",
```

Comment:
@noahtalerman  @rachaelshaw I noticed the existing API returns all available information for the policy. Is this a bug?
```
"failing_policies": [
            {
                "id": 3,
                "name": "Policy 1",
                "query": "SELECT 0;",
                "critical": false,
                "description": "This policy checks if FileVault (disk encryption) is enabled.",
                "author_id": 1,
                "author_name": "Gandalf",
                "author_email": "foo@example.com",
                "team_id": null,
                "resolution": "As an IT admin, turn on disk encryption in Fleet.",
                "platform": "darwin",
                "created_at": "2024-03-16T21:19:39Z",
                "updated_at": "2024-03-19T19:55:11Z",
                "response": "fail"
            }
        ]
```

Reply:
we're also returning additional information in `vulnerable_software` (CVSS etc...), do we want the response to conform to what we have here?

Reply:
> the existing API returns all available information for the policy. Is this a bug?

@mostlikelee yes, I think that's a bug. We want the response to reflect the response here.

Great catch.

@rachaelshaw please correct me if I'm wrong.

Reply:
@mostlikelee @noahtalerman yeah device health was intended to have just the essentials

Reply:
Thanks all!  Just want to doublecheck before I remove information from the current API response.